src/NFun.Fuspec.TestHandler/FuspecTestCasesRunner.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Diagnostics;
+using Nfun.Fuspec.Parser.Model;
+using NFun.Fuspec.TestHandler.Models;
+using NFun.ParseErrors;
+using NFun.Runtime;
+
+namespace NFun.Fuspec.TestHandler
+{
+    public static class FuspecTestCasesRunner
+    {
+        public static BaseFuspecTestCaseResult RunFuspecTest(FuspecTestCase testCase)
+        {
+            var stopwatch = Stopwatch.StartNew();
+            return ParseTestResult(testCase, stopwatch);
+        }
+
+        private static BaseFuspecTestCaseResult ParseTestResult(FuspecTestCase testCase, Stopwatch stopwatch)
```

Comment:
Why do you pass stopwatch through the argument? Why don't you create and finish it in the same place?

var sw = Stopwatch.StartNew()
DoSomeShit()
sw.Stop()